src/sftp.rs

```diff
@@ -146,6 +147,7 @@ impl Sftp {
                 options.get_max_pending_requests(),
                 auxiliary,
                 options.get_tokio_compat_file_write_limit(),
+                Handle::current(),
```

Comment:
I would like this to be put into `Sftp::connect` to avoid monomorphization generating duplicate code.

Reply:
changed in new commit